client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -33,17 +30,14 @@ var MOCK_SITE = {
 	options: {}
 };
 
-var expect = chai.expect;
-
-chai.use( sinonChai );
-
 describe( 'EditorGroundControl', function() {
+	let EditorGroundControl;
+
+	useMockery();
+	useFakeDom();
 	this.timeout( 10 * 1000 );
```

Comment:
Thanks for converting to Enzyme ;)
Most likely extended timeout time was introduced only to avoid Circle CI failures. @blowery set 10s default timeout for Circle CI env so we no longer need to set it for individual test suites. We can now also easily catch slow tests using Circle CI integration with JUnit output style. See:

<img width="1300" alt="screen shot 2016-04-07 at 07 49 24" src="https://cloud.githubusercontent.com/assets/699132/14341877/9774f3fe-fc95-11e5-9162-9ea9635c27bc.png">

Also `Artifacts` tab is very interesting:
<img width="858" alt="screen shot 2016-04-07 at 07 52 08" src="https://cloud.githubusercontent.com/assets/699132/14341887/b060820c-fc95-11e5-8529-3c9af830eec8.png">

You can download detailed test output there.
 
BTW, I have never seen different message than those about token-field component and its execution time is also very stable - 0.7-ish s.